var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -211,6 +211,9 @@ def configure_args(self):
             else:
                 args.append("--enable-{0}=no".format(fabric))
 
+        if self.spec.satisfies("+cuda"):
```

Comment:
Might as well add the logic for other HMEM interfaces in this manner?

Reply:
Ping me in a comment when this PR is ready to merge then.

Reply:
Thanks @rajachan!

I am happy to do the same for ROCm, but would it be acceptable to do so as a separate PR? If that's ok, I will try to get to it soon.

Reply:
Sounds good. @tldahlgren this is ready to go then.